---
# Feel free to add content and custom Front Matter to this file.
# To modify the layout, see https://jekyllrb.com/docs/themes/#overriding-theme-defaults

layout: default
theme: minima
---

![DwM](images/prof_pic.jpg){: style="float: left; height: 180px; margin-right: 12px; border-radius:42%;"}

## **David W. McKellar**  

```
PhD Candidate  
Biomedical Engineering  
Cornell University  
dwm269 [at] cornell [dot] edu  
```  

-----------

[**google scholar**](https://scholar.google.com/citations?user=Hta5xCcAAAAJ&hl=en&oi=ao) |
[**github.com/mckellardw**](https://github.com/mckellardw) |
[**twitter: @dwmckellar**](https://twitter.com/dwmckellar)

-----------

## **About**:
I'm a PhD candidate in the Biomedical Engineering Department at Cornell, co-advised by [Ben Cosgrove](https://cosgrovelab.bme.cornell.edu/) and [Iwijn De Vlaminck](https://devlaminck.bme.cornell.edu/). I work on computational and molecular strategies to understand single-cell and spatial biology, with particular interests in stem cells and noncoding RNAs.

<!-- ## **Highlights**:
+ My newest first author preprint on a method we call Spatial Total RNA-Sequencing (STRS). STRS enables spatial mapping of noncoding and other non-polyadenylated RNAs in tissue slices: [link](https://www.biorxiv.org/content/10.1101/2022.04.20.488964v1)
+ My recent first author paper on building a single-cell reference dataset for skeletal muscle regeneration, published in Communications Biology: [link](https://www.nature.com/articles/s42003-021-02810-x)
+ Highlight from the BME Department at Cornell: [link](https://www.bme.cornell.edu/spotlights/david-mckellar-phd-student) -->
