### **David W. McKellar**

My thesis work has centered on finding new ways to squeeze information out of transcriptomics data.

## 1. Exploring stem cell biology with large-scale single cell transcriptomics
Stem cells are inherently lowly abundant in tissues. To characterize transient states of skeletal muscle stem cells, we integrated single-cell RNA-sequencing data from >365k single cells and nuclei and mined the resulting compendium for committed muscle stem cells.

This work resulted in a first author manuscript in [*Communications Biology*](https://www.nature.com/articles/s42003-021-02810-x).

![scMuscle](images/thesis_summary/scMuscle_umap_harmony_vBlack.png)


## 2. Expanding spatial transcriptomics into the world of noncoding RNAs
Current spatial transcriptomics technologies only capture RNAs which have poly(A) tails and thus overlook the wealth of information in the non-A-tailed portion of the transcriptome. **Spatial Total RNA-Sequencing (STRS)** circumvents this problem by polyadenylating RNAs in situ prior to hybridization.

This work is detailed in a [preprint](https://www.biorxiv.org/content/10.1101/2022.04.20.488964v1) and is currently under review.

![scMuscle](images/thesis_summary/STRS_v2_DarkMode.png)
