---
layout: page
title: Publications
permalink: /publications/
---

<div style="display: flex; gap: 10px;">
  <a href="https://scholar.google.com/citations?user=Hta5xCcAAAAJ&hl=en&oi=ao" style="text-decoration: none; padding: 10px; background-color: #bbbbbb; border-radius: 5px; display: flex; align-items: center;">
    <img src="https://upload.wikimedia.org/wikipedia/commons/2/28/Google_Scholar_logo.png" alt="Google Scholar" style="height: 20px;">
  </a>
  <a href="https://pubmed.ncbi.nlm.nih.gov/?term=David+McKellar%5BAuthor%5D" style="text-decoration: none; padding: 10px; background-color: #bbbbbb; border-radius: 5px; display: flex; align-items: center;">
    <img src="https://upload.wikimedia.org/wikipedia/commons/f/fb/US-NLM-PubMed-Logo.svg" alt="PubMed" style="height: 20px;">
  </a>
</div>

## **Highlighted**
***High Resolution Spatial Mapping of Microbiome-Host Interactions via in situ Polyadenylation and Spatial RNA Sequencing.***  
I Ntekas^, L Takayasu^, **DW McKellar^**, B Grodner, M Hinchman, C Holdener, P Schweitzer, M Sauthoff, Q Shi, L Brito, I de Vlaminck  
[[**bioRxiv, 2024**]](https://doi.org/10.1101/2024.11.18.624127)
[[**github**]](https://github.com/ntekasi/microSTRS)
[[**pdf**]](https://mckellardw.github.io/pdfs/manuscripts/Ntekas_et_al_bioRxiv_2024.pdf)

***Spatial mapping of the total transcriptome by in situ polyadenylation.***  
**DW McKellar**, M Mantri, M Hinchman, JSL Parker, P Sethupathy, BD Cosgrove, I de Vlaminck  
[[**Nature Biotechnology, 2023**]](https://www.nature.com/articles/s41587-022-01517-6)
[[**bioRxiv, 2022**]](https://doi.org/10.1101/2022.04.20.488964)
[[**github**]](https://github.com/mckellardw/STRS)
[[**pdf**]](https://mckellardw.github.io/pdfs/manuscripts/McKellar_et_al_NatureBiotechnology_2022.pdf)  
[[**Research Briefing**]](https://www.nature.com/articles/s41587-022-01562-1)

***Large-scale integration of single-cell transcriptomic data captures transitional progenitor states in mouse skeletal muscle regeneration.***  
**DW McKellar**, LD Walter, LT Song, M Mantri, MFZ Wang, I de Vlaminck, BD Cosgrove  
[[**Communications Biology, 2021**]](https://doi.org/10.1038/s42003-021-02810-x)
[[**bioRxiv, 2020**]](https://www.biorxiv.org/content/10.1101/2020.12.01.407460v2)
[[**github**]](https://github.com/mckellardw/scMuscle)
[[**pdf**]](https://mckellardw.github.io/pdfs/manuscripts/McKellar_et_al_Communications_Biology_2021.pdf)

-----

## **Full list**
### *Preprints*
***High Resolution Spatial Mapping of Microbiome-Host Interactions via in situ Polyadenylation and Spatial RNA Sequencing.***  
I Ntekas^, L Takayasu^, **DW McKellar** B Grodner, M Hinchman, C Holdener, P Schweitzer, M Sauthoff, Q Shi, L Brito, I de Vlaminck  
[[**bioRxiv, 2024**]](https://doi.org/10.1101/2024.11.18.624127)
[[**github**]](https://github.com/ntekasi/microSTRS)
[[**pdf**]](https://mckellardw.github.io/pdfs/manuscripts/Ntekas_et_al_bioRxiv_2024.pdf)

### *Published*
***Pre-ciliated tubal epithelial cells are prone to initiation of high-grade serous ovarian carcinoma.***  
A Flesken-Nikitin, CQ Ralston, DJ Fu, AJ De Micheli, DJ Phuong, BA Harlan, CS Ashe, AP Armstrong, **DW McKellar**, S Ghuwalewala, LH Ellison, JC Schimenti, BD Cosgrove, AY Nikitin  
[[**Nature Communications, 2024***]](https://www.nature.com/articles/s41467-024-52984-1)
[[**bioRxiv, 2023**]](https://doi.org/10.1101/2023.12.12.571315)
[[**pdf**]](https://mckellardw.github.io/pdfs/manuscripts/Ralston_Nature_Communications_2024.pdf)

***Hedgehog signaling acts as cell fate determinant during adult tissue repair.***  
AM Norris, CD Johnson, LY Zhou, A Appu, **DW McKellar**, BD Cosgrove, D Kopinke  
[[**Nature Communications, 2023**]](https://www.nature.com/articles/s41467-023-39506-1)
[[**bioRxiv, 2022**]](https://doi.org/10.1101/2022.08.15.504012)
[[**pdf**]](https://mckellardw.github.io/pdfs/manuscripts/Norris_NatureCommunications_2023.pdf)

***Spatial mapping of the total transcriptome by in situ polyadenylation.***  
**DW McKellar**, M Mantri, M Hinchman, JSL Parker, P Sethupathy, BD Cosgrove, I de Vlaminck  
[[**Nature Biotechnology, 2022**]](https://www.nature.com/articles/s41587-022-01517-6)
[[**bioRxiv, 2022**]](https://doi.org/10.1101/2022.04.20.488964)
[[**github**]](https://github.com/mckellardw/STRS)
[[**pdf**]](https://mckellardw.github.io/pdfs/manuscripts/McKellar_et_al_NatureBiotechnology_2022.pdf)

***Spatiotemporal transcriptomics reveals pathogenesis of viral myocarditis.***  
M Mantri, MM Hinchman, **DW McKellar**, Z Wang, MF L Parker, I de Vlaminck  
[[**Nature Cardiovascular Research, 2022**]](https://www.nature.com/articles/s44161-022-00138-1)
[[**bioRxiv, 2021**]](https://doi.org/10.1101/2021.12.07.471659)
[[**pdf**]](https://mckellardw.github.io/pdfs/manuscripts/Mantri_NatCardioResearch_2022.pdf)

***A Tead1-Apelin axis directs paracrine communication from myogenic to endothelial cells in skeletal muscle.***  
U Lee, P Stuelsatz, S Karaz, **DW McKellar**, J Russeil, M Deak, I de Vlaminck, C Lepper, B Deplancke, BD Cosgrove, JN Feige  
[[**iScience, 2022**]](https://doi.org/10.1016/j.isci.2022.104589)
[[**pdf**]](https://mckellardw.github.io/pdfs/manuscripts/Lee_et_al_iScience_2022.pdf)

***Large-scale integration of single-cell transcriptomic data captures transitional progenitor states in mouse skeletal muscle regeneration.***  
**DW McKellar**, LD Walter, LT Song, M Mantri, MFZ Wang, I de Vlaminck, BD Cosgrove  
[[**Communications Biology, 2021**]](https://doi.org/10.1038/s42003-021-02810-x)
[[**bioRxiv, 2020**]](https://www.biorxiv.org/content/10.1101/2020.12.01.407460v2)
[[**github**]](https://github.com/mckellardw/scMuscle)
[[**pdf**]](https://mckellardw.github.io/pdfs/manuscripts/McKellar_et_al_Communications_Biology_2021.pdf)

***Uncovering transcriptional dark matter via gene annotation independent single-cell RNA sequencing analysis.***  
MFZ Wang, M Mantri, SP Chou, GJ Scuderi, **DW McKellar**, JT Butcher, CG Danko, I de Vlaminck  
[[**Nature Communications, 2021**]](https://www.nature.com/articles/s41467-021-22496-3)
[[**bioRxiv, 2020**]](https://www.biorxiv.org/content/10.1101/2020.07.31.229575v1.full)
[[**github**]](https://github.com/fw262/TAR-scRNA-seq)
[[**pdf**]](https://mckellardw.github.io/pdfs/manuscripts/Wang_et_al_Nature_Communications_2021.pdf)

***Spatiotemporal single-cell RNA sequencing of developing chicken hearts identifies interplay between cellular differentiation and morphogenesis.***  
M Mantri, GJ Scuderi, R Abedini-Nassab, MFZ Wang, **DW McKellar**, H Shi, B Grodner, JT Butcher, I de Vlaminck  
[[**Nature Communications, 2021**]](https://www.nature.com/articles/s41467-021-21892-z)
[[**bioRxiv, 2020**]](https://www.biorxiv.org/content/10.1101/2020.05.03.065102v1)
[[**pdf**]](https://mckellardw.github.io/pdfs/manuscripts/Mantri_et_al_Nature_Communications_2021.pdf)
